package.json

```diff
@@ -22,7 +22,7 @@
     "build": "craco build",
     "serve": "serve build -l 3000",
     "lint": "yarn eslint --ignore-path .gitignore --cache --cache-location node_modules/.cache/eslint/ .",
-    "typecheck": "tsc --noEmit",
+    "typecheck": "tsc",
```

Comment:
Is this intended? 
> The noEmit option tells TypeScript that we only want to run type checking and do not want the compiler to output any transpiled code

Reply:
Yes! noEmit is included in our tsconfig.